packages/lab/src/pill/internal/PillBase.css

```diff
@@ -199,3 +236,13 @@
   padding-right: 0;
   width: var(--pill-checkbox-size);
 }
+
+.uitk-light .backwardsCompat.uitkPill {
+  --pill-background: var(--uitk-color-grey-40);
```

Comment:
This will fail the style lint check when move to core. Need to think about how to deal with these.